### ISSUE TITLE
Resource are shared only for one test

### ISSUE THREAD

**Issue by Rzou28** [kangasta/rf-shared-resources#3]
Hi,

I'm using rf-shared-resources to import my created library.
The problem I encounter is : 
When I run the robot command on a folder containing multiple suites (.robot), the library is imported successfully for the first executed suite, but it seems like the same library is not being available for the second one (which is called the same way).

I tried to run the pabot command (with one or multiple processes) on the same folder containing the same multiple suites, and everything works fine.

Is there a restriction on the number of suites you can import your library into using the robot command (which is the default one for RF)? I am the only with this issue?

Regards,

**Comment by kangasta** [comment]
Hi!

Thanks for using the library and reporting this bug.

I was able to replicate the issue as well: Seems that the `__init__()` method of the library class is not called again after the library is first imported, because Robot Framework caches libraries internally (See https://github.com/robotframework/robotframework/issues/2106)

I added `import_library` listener that should hande resource loading on second and further imports. See #4 for details.
